tools/file_packager.py

```diff
@@ -568,7 +574,7 @@ def was_seen(name):
       var indexedDB = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
       var IDB_RO = "readonly";
       var IDB_RW = "readwrite";
-      var DB_NAME = 'EM_PRELOAD_CACHE';
+      var DB_NAME = "''' + indexeddb_name + '''";
```

Comment:
are these quotes right? shouldn't it be `"'" + indexeddb_name + "'"` or `'\'' + indexeddb_name + '\''`?